packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -64,4 +64,5 @@ export interface FeatureToggles {
   internationalization?: boolean;
   topnav?: boolean;
   customBranding?: boolean;
+  azTemplateVars?: boolean;
```

Comment:
did you manually edit this file, or just forget to commit in `pkg/services/featuremgmt/registry.go`?

Reply:
> DO NOT EDIT DIRECTLY!

who does that...